fineract-provider/src/main/java/org/apache/fineract/cob/common/InitialisationTasklet.java

```diff
@@ -41,11 +47,17 @@ public class InitialisationTasklet implements Tasklet {
 
     @Override
     public RepeatStatus execute(@NotNull StepContribution contribution, @NotNull ChunkContext chunkContext) throws Exception {
+        HashMap<BusinessDateType, LocalDate> businessDates = ThreadLocalContextUtil.getBusinessDates();
```

Comment:
Not a great problem, but it would be nice to not use directly the ThreadLocalContextUtil, rather fetch it from DateUtils, which can fetch it from the ThreadLocalContextUtil. Its just nicer to fetch it through the same object (DateUtils)